_includes/head.html

```diff
@@ -15,6 +15,7 @@
     <!-- Custom Fonts -->
     <link href='https://fonts.googleapis.com/css?family=Nunito:300italic,400italic,600italic,700italic,800italic,400,300,600,700,800' rel='stylesheet' type='text/css'>
     <link href='https://fonts.googleapis.com/css?family=Lato:400,300,300italic,400italic,700,700italic,900,900italic' rel='stylesheet' type='text/css'>
+    <link rel="stylesheet" href="font-awesome/css/font-awesome.min.css" type="text/css">
```

Comment:
what is this awesome font thing 😅

Reply:
apparently for icons and vectors. it's in the starter theme, I'm just restoring it

Reply:
ah yep, i deleted it lol